### DISCUSSION TITLE
There is no data in the notification list, but there are corner markers.

### DISCUSSION THREAD

**Discussion by aimuz** [community/community#111157] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

<img width="1059" alt="图片" src="https://github.com/community/community/assets/17666471/2cc27aa2-1af0-4cbb-9226-bbb9274f739b">

In the diagram you can see the incorrectly displayed number 1

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @aimuz ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by aimuz** [comment]
But apparently the problem is a GitHub bug. 🧐

**Comment by aimuz** [comment, marked as answer]
This solves my problem, the token needs to be created here: https://github.com/settings/tokens/new.

```bash
export now=$(date +%Y-%m-%dT%H:%M:%SZ)
curl -H "Authorization: bearer $TOKEN" -X PUT -H "Accept: application/vnd.github.v3+json" https://api.github.com/notifications -d '{"last_read_at":"'"$now"'"}'
```

**Reply by nosqldev** [reply]
only this solution worked

**Reply by faridvatani** [reply]
I tested this solution using Notifications Access only and it worked. Thank you. 🙏🏼

**Comment by queenofcorgis** [comment]
We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.
